### ISSUE TITLE
Update Server: Modify or Add Labware Defs via Update Server 

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#1550]
As a user, I would like to be able to update labware defs via the robot update server

## Acceptance Criteria
- [ ] User able to update labware defs via update server endpoint
- [ ] Robot automatically updates offset such that calibration remains the same
-- Compare new labware defs to old labware defs, apply delta to offsets

**Comment by btmorr** [comment]
Blocked on #1208

**Comment by btmorr** [comment]
Labware definition system design has changed, so this ticket is no longer relevant
